ui/src/admin/components/EmptyRow.js

```diff
@@ -1,12 +1,11 @@
 import React, {PropTypes} from 'react'
 
-const EmptyRow = ({tableName}) => (
+const EmptyRow = ({tableName}) =>
   <tr className="table-empty-state">
     <th colSpan="5">
       <p>You don't have any {tableName},<br />why not create one?</p>
```

Comment:
Is this file still valid? The syntax highlighting on GitHub is concerning:
<img width="834" alt="screen shot 2017-06-07 at 11 30 16 am" src="https://user-images.githubusercontent.com/6403018/26889730-b24f9826-4b74-11e7-9536-c6a95f489790.png">